site/content/en/docs/Getting Started/edit-first-gameserver-go.md

```diff
@@ -87,6 +87,22 @@ kubectl create -f gameserver.yaml
 kubectl apply -f gameserver.yaml
 ```
 
+{{< alert title="Note" color="info">}}
+If you made changed `main.go` again and want to apply the changes to the new game servers, then you also need 
+  to modify the `gamerserver.yaml` file's `imagePullPolicy` to be `Always`, or else the new game servers might
+  only pull a cached version of the image, which doesn't have the new changes.
+  ```yaml
+  spec:
+  containers:
+  - name: simple-game-server
+    image: ${REGISTRY}/simple-game-server:${TAG}
+    imagePullPolicy: Always
+  ```
+
+Alternatively, you can also manually increment the `version` field in the `Makefile` file and change the `TAG` variable accordingly.
+However this is more cumbersome, and is not encouraged.
```

Comment:
Why not encouraged?

Reply:
Oh I guess it's just less preferred during development period because it is more cumbersome. Added some more explanations and changed the wording a bit.